modules/oe_content_news/oe_content_news.post_update.php

```diff
@@ -23,3 +25,28 @@ function oe_content_news_post_update_00001_update_field_labels(array &$sandbox):
     $field_config->save();
   }
 }
+
+/**
+ * Create oe_reference field in the news content type.
+ */
+function oe_content_news_post_update_00002(): void {
+  $storage = new FileStorage(drupal_get_path('module', 'oe_content_news') . '/config/post_updates/00002_create_fields');
+  \Drupal::service('config.installer')->installOptionalConfig($storage);
+}
+
+/**
+ * Update news node form display.
+ */
+function oe_content_news_post_update_00003(): void {
```

Comment:
I don't see the point in splitting this, the way I see it there should be 1 updated per epic per entity type, I know it was requested in 3119; @ademarco what are the guidelines here?

Reply:
it's ok to split it per "type of task", so to say, so that if one of these fail in production we know where it failed more precisely.